builtin/game/statbars.lua

```diff
@@ -175,6 +192,20 @@ function core.hud_replace_builtin(hud_name, definition)
 		return true
 	end
 
+	if hud_name == "minimap" then
+		bar_definitions.minimap = definition
+
+		for name, ids in pairs(hud_ids) do
+			local player = core.get_player_by_name(name)
+			if player and ids.id_minimap then
+				player:hud_remove(ids.id_minimap)
+				ids.id_minimap = nil
+				update_builtin_statbars(player)
+			end
+		end
+		return true
+	end
```

Comment:
This is unfortunately broken because of this check above, which only permits type "statbar" HUD elements:

https://github.com/minetest/minetest/blob/95fc63a70b09ead8486fe5857adb65a174eeec58/builtin/game/statbars.lua#L158-L161

Maybe we can just remove it completely? It's quite obvious if the modder messes up the HUD this way.

Reply:
I removed the statbar check.

Reply:
The statbar check is still there.

Reply:
Sorry, I added the wrong thing, it's gone now.